modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile Registry to access product Devfiles
+
+You can use the Devfiles in the  in the *+Add* flow of the *Developer* perspective to create an application. The Devfile registry provides a complete integration with the public community registry and the product registry. A devfile is a portable YAML file that describes your development environment without needing to configure it from scratch. Using the Devfile registry, you can use a pre-configured devfile to create an application.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and from the Developer Catalog tile, click *All Services* to view all the available services in the Developer Catalog.
```

Comment:
I think there is some scope for applying minimalism principles here. See if you can rewrite it in the "... **UI_elem_1** > **UI_elem_2** > Do something" format, without distorting the meaning. See [this slide](https://docs.google.com/presentation/d/e/2PACX-1vQUwdfpeqHl07nDyaE2t04zIhRQ4v_q9pkeyszUWgoMAwuizD0M4YbnWxVRwSTHw_vjNK5sYpwGzxaA/pub?start=false&loop=false&delayms=3000&slide=id.g88e33f052a_2_149) and the next few.